app/_includes/md/about-plugins.md

```diff
@@ -104,7 +84,10 @@ Kong provides PDKs for two languages: Lua and Go. Both of these PDKs are sets
 of functions that a plugin can use to facilitate interactions between plugins
 and the core (or other components) of Kong.
 
-To start creating your own plugins, check out the PDK documentation:
+To start creating your own plugins, review the [Getting Started documentation](/gateway/latest/plugin-development/get-started/).
+
+For more information about the PDK, review the following documentation: 
+
 * [Plugin Development Guide](/gateway/latest/plugin-development/)
 * [Plugin Development Kit reference](/gateway/latest/plugin-development/pdk/)
 * [Other Language Support](/gateway/latest/plugin-development/pluginserver/go/)
```

Comment:
We should add a bit more info here - maybe a list/links of plugin development languages we support, and the performance page? 
https://docs.konghq.com/gateway/latest/plugin-development/pluginserver/performance/